oracle skript

ihatevim opened the pull request:
auto heal below slider threshold for hp
auto ks with e
auto heal ulted target
auto combo to never heal enemy

Commit:
oracle skript

ok

ghost commented:
i have ult in the supported abilities but its not doing anything, feel free to remove that

Commit:
removed useless icon

ghost commented:
also its intentionally interrupting the channel with the orbwalk (to purge the heal) so dont remove that

Commit:
distance check

Commit:
hero toggler

thanks idcnoob

Commit:
StormSpirit

-smart mana management
-sliders for mana post chase ult
-auto killsteal with async block
-dynamic combo based on spells available and mana

Commit:
ult tweaks

just some small tweaks

Commit:
better movement + aghs

will stay right on top of your target with this one and aghanims support

Commit:
mr check/spell correct

Commit:
added ks checks

more ks checks

Commit:
default maxdist

increased to 1000

Commit:
removed useless references

Commit:
mana logic for q

will now q if it cannot consequentially cast w and q due to mana
insufficiency

Commit:
minor tweaks

Commit:
improved speed

Commit:
faster

another .1 faster

Commit:
lol woops

that should have been there a while ago

Commit:
fix double ult

wont do that anymore

Commit:
improved chase distance

minor improvement to chase
